styles/Hero.module.css

```diff
@@ -0,0 +1,68 @@
+.container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    padding-bottom: var(--spacing-unit);
+}
+
+.hero_container {
+    margin-top: 319.5px;
+    display: flex;
+    flex-direction: column;
+    gap: 29.5px;
+}
+
+.welcome_box {
+    color: var(--primary-blue);
+    padding: 8px;
+}
+
+.welcome_text {
+    font-size: 50px;
+    height: 58px;
+}
+
+.intro_text {
+    padding: 15% 0px 0px 0px;
```

Comment:
Please use CSS vars from globals

Reply:
No variable in the CSS globals that reference this box position

Reply:
fixed